user/zerotier.rst

```diff
@@ -0,0 +1,131 @@
+ZeroTier tunnel automation
+--------------------------
+
+In this guide we will explore how to set up the automatic management
+of `ZeroTier tunnels. <https://www.zerotier.com/>`_
+
+.. contents:: **Table of Contents**:
+   :backlinks: none
+   :depth: 3
+
+How to setup ZeroTier Tunnels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the procedure described below to setup
+ZeroTier tunnels on your devices.
+
+**Note:** This example uses **Shared systemwide (no organization)**
+option as the organization for VPN server and VPN client template.
+You can use any organization as long as VPN server, VPN client
+template and Device has same organization.
+
+1. Configure Self-Hosted ZeroTier Network Controller
+####################################################
+
+If you haven't already set up a self-hosted Zerotier
```

Comment:
Can we please mention here that we don't support the Zerotier SAAS offering yet and you *need* the self-hosted version.

Reply:
Sure!